README.md

```diff
@@ -19,10 +19,9 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 
 ## Documentation
 
-- [User Documentation](https://www.odoo.com/documentation/14.0/applications.html) - Official user documentation
-- [Development Documentation](https://www.odoo.com/documentation/14.0/) - Official development documentation
-- [Documentation for Install and Maintain](https://www.odoo.com/documentation/14.0/administration.html) - Documentation for Install and Maintain
+- [Product Documentation](https://www.odoo.com/documentation) - Official Documentation - User Docs, Install and Maintain, Developer, Contributing
 - [Data Model Documentation for V8](http://useopenerp.com/v8) with UML diagrams.
+- [Data Model Documentation] (https://www.odoo.com/documentation) generated by ERBuilder
```

Comment:
Where is this?

Reply:
Sorry, the link should be (https://soft-builder.com/en/docs/Odoo/index.html)

Reply:
I find it a good resource, but it is incomplete, many tables are missing.

I recommend adding it as an additional line and keeping the one from v8

Reply:
I agree